sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -48,6 +50,11 @@ export interface ConnectionContext extends ConnectionContextBase {
    * the underlying amqp connection for the EventHub Client.
    */
   managementSession?: ManagementClient;
+
```

Comment:
Can we have doc comments for the 3 things being added to the ConnectionContext?
Helps in maintaining the pattern as all other properties have the docs and also clarifies the first confusion anyone would have which is "what is the difference between connection closing and disconnecting"?